### PR TITLE
Logical operator

### PR DESCRIPTION
**Pull request by komal7141**

### CODE REVIEW

File: .cpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+Int main()
+{
+    Int m1,m2,m3,m4,m5,per;
+    Printf("enter marks in five subjects");
+    Scanf("%d%d%d%d%d",&m1,&m2,&m3,&m4,&m5);
+
+     Per=(m1+m2+m3+m4+m5)/500*100;
+     
+    If(per>=60)
```
Comment (sarvesh-0) [inline .cpp:10]: Is this code working? You have Per in line 8, and at rest of the places its per.
Could you please confirm?

File: .cpp [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+          Printf("first division\n");
+     
+    If((per>=50)&&(per<50))
```
Comment (sarvesh-0) [inline .cpp:13]: Can you please use || and ! as well in this code so that it could become more informative.

Comment (komal7141) [reply .cpp:13]: Ok sir I'll try to do again and will add some more stuff here.

**Review by Anjali-1801 — COMMENTED** [review]
1. The keywords must be in small letters, like int, printf, scanf, if, etc.
2. Also, at one place 'Per' is used and at others 'per'. 
3. Only 'if' is used, you need to use 'else if' as well.

**Pull request merged**